content/800-data-platform/300-cloud-projects/200-platform/200-projects.mdx

```diff
@@ -188,69 +188,52 @@ See [Usage](/data-platform/cloud-projects/platform/organizations#usage) for deta
 
 ## Pulse
 
-Within each Cloud Project you can set up and configure Pulse, Prisma's manage [change data capture](https://en.wikipedia.org/wiki/Change_data_capture) (CDC) service.
+Within each Cloud Project, you can set up and configure Pulse, Prisma's manage [change data capture](https://en.wikipedia.org/wiki/Change_data_capture) (CDC) service.
 
-### Configure Pulse
+### Enable Pulse
```

Comment:
```suggestion
### Set up Pulse
```
Maybe to avoid the duplication in headlines?

Reply:
I like this suggestion, thanks @janpio This also aligns well with how it translates on the platform side.

You have to **set up** Pulse, which requires;
- preparing your database
- and enabling Pulse